### ISSUE TITLE
batch move/delete/copy on search results

### ISSUE THREAD

**Issue by wherejumpout** [siyuan-note/siyuan#8539]
### In what scenarios do you need this feature?

when searching a topic, siyuan returns a list of documents and/or blocks.
I want to move several documents to a directory
Now I can only click on one document and open it, then move it to the directory.
Then I search the topic again, find the second target document and move it to the directory.
So and so forth.

### Describe the optimal solution

cmd + click on multiple documents, and move it in a batch.

As in the doc tree.

![image](https://github.com/siyuan-note/siyuan/assets/128914992/5ef9e5a2-7f11-4d46-9ccd-f74e1701d219)

**Comment by Vanessa219** [comment]
Copy does not support batch operations.

**Comment by 88250** [comment]
After analysis, there are some problems with moving or deleting the search results (if you only move the document block, there is no problem, but moving some sub-blocks, such as the leaf block under the list item, there will be problems), so this requirement cannot be implemented at present.

Anyway, thank you for making this request.

**Comment by wherejumpout** [comment]
yes, what i need is moving the documents.  Hope this feature will be present in some furthur version

**Comment by Vanessa219** [comment]
However, not all documents are selected, and other block types may be selected.
